docs/getting-started-kubernetes.rst

```diff
@@ -16,7 +16,1259 @@
+      containers:
+      - name: appcntr1
+        image: <image>
+        imagePullPolicy: IfNotPresent
+        securityContext:
+          capabilities:
+            add: ["IPC_LOCK"]
+        resources:
+          requests:
+            nvidia.com/sriov_resource: '1'
+          limits:
+            nvidia.com/sriov_resource: '1'
+        command:
+        - sh
+        - -c
+        - sleep inf
+
+---------------------------------------------------------------------------
+SR-IOV Network Operator Deployment – Parallel Node Configuration for SR-IOV
+---------------------------------------------------------------------------
+
+.. warning:: This is a Tech Preview feature, which is supported only for Vanilla Kubernetes deployments with SR-IOV Network Operator.
+
+To apply SriovNetworkNodePolicy on several nodes in parallel, specify the ``maxParallelConfiguration`` option in the SriovOperatorConfig CRD:
+
+.. code-block:: bash
+
+    kubectl patch sriovoperatorconfigs.sriovnetwork.openshift.io -n network-operator default --patch '{ "spec": { "maxParallelNodeConfiguration": 0 } }' --type='merge'
+
+--------------------------------------------------------------------------
+SR-IOV Network Operator Deployment – Parallel NIC Configuration for SR-IOV
+--------------------------------------------------------------------------
+
+.. warning:: This is a Tech Preview feature, which is supported only for Vanilla Kubernetes deployments with SR-IOV Network Operator.
+
+To apply SriovNetworkNodePolicy on several nodes in parallel, specify the ``featureGates`` option in the SriovOperatorConfig CRD:
+
+.. code-block:: bash
+
+    kubectl patch sriovoperatorconfigs.sriovnetwork.openshift.io -n network-operator default --patch '{ "spec": { "featureGates": { "parallelNicConfig": true  } } }' --type='merge'
+
+---------------------------------------------------------------------------
+SR-IOV Network Operator Deployment – Parallel Node Configuration for SR-IOV
+---------------------------------------------------------------------------
+
+To apply SriovNetworkNodePolicy on several nodes in parallel, specify the ``configurationMode`` option in the SriovOperatorConfig CRD:
+
+.. code-block:: bash
+
+    kubectl patch sriovoperatorconfigs.sriovnetwork.openshift.io -n network-operator default --patch '{ "spec": { "configurationMode": "systemd"} }' --type='merge'
+
+-------------------------------------------------------------
+Network Operator Deployment with an SR-IOV InfiniBand Network
+-------------------------------------------------------------
+
+Network Operator deployment with InfiniBand network requires the following:
+
+* MLNX_OFED and OpenSM running. OpenSM runs on top of the MLNX_OFED stack, so both the driver and the subnet manager should come from the same installation. Note that partitions that are configured by OpenSM should specify defmember=full to enable the SR-IOV functionality over InfiniBand. For more details, please refer to this article.
```

Comment:
Missing link in `this article`
https://docs.mellanox.com/display/MLNXOFEDv51258060/OpenSM